doc/Installation/Installation-CentOS-7-Apache.md

```diff
@@ -69,6 +79,14 @@ lower_case_table_names=0
 Ensure date.timezone is set in php.ini to your preferred time zone.  See http://php.net/manual/en/timezones.php for a list of supported timezones.  Valid examples are: "America/New_York", "Australia/Brisbane", "Etc/UTC".
 
     vi  /etc/php.ini
+    
+or
+
+    vi /etc/opt/rh/rh-php72/php.ini
+
+When PHP is configured with open_basedir, be sure to allow the following paths for LibreNMS to work:
+
+    php_admin_value[open_basedir] = /opt/librenms:/dev/urandom:/usr/sbin/fping:/usr/sbin/fping6:/usr/bin/snmpgetnext:/usr/bin/rrdtool:/usr/bin/snmpwalk:/usr/bin/snmpget:/usr/bin/snmpbulkwalk:/usr/bin/traceroute
```

Comment:
This isn't covering all executables used, just the most common ones.

Reply:
Well those are the executables I could find and that make my librenms setup run. Pleae give me a list of missing ones, I'd be happy to improve the docs.